### PR TITLE
blaze: fixup

### PR DESCRIPTION
**Pull request by sevki**
use + to add to arrays together

### CODE REVIEW

File: sys/src/9/amd64/BUILD [hunk 1]
```diff
@@ -490,10 +489,7 @@ cc_binary(
         "-fno-builtin",
         "-mcmodel=small",
     ],
-    deps=[
-	LIBS,
-	KLIBS,
-    ],
+    deps=LIBS + KLIBS,
```
Comment (rminnich) [inline sys/src/9/amd64/BUILD:492]: This is nice. Can you still use an array or does it have to be the + form?

**Comment by rminnich** [comment]
LGTM

File: sys/src/9/amd64/BUILD (hunk 1 above)
Comment (sevki) [inline sys/src/9/amd64/BUILD:492]: it has to be in + form. python grammar

**Comment by sevki** [comment]
ron if you merge this it will update your PR

**Comment by sevki** [comment]
@rminnich  if you merge this it will update your PR

**Pull request merged**